src/components/[guild]/activity/ActivityLogAction/components/FormTag.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  Button,
+  Circle,
+  forwardRef,
+  HStack,
+  Tag,
+  TagProps,
+  TagRightIcon,
+  Text,
+  useColorModeValue,
+} from "@chakra-ui/react"
+import { DotsThreeVertical, Funnel, PencilSimpleLine } from "phosphor-react"
+import useSWRImmutable from "swr/immutable"
+import { ActivityLogType } from "../../ActivityLogContext"
+import {
+  FILTER_NAMES,
+  useActivityLogFilters,
+} from "../../ActivityLogFiltersBar/components/ActivityLogFiltersContext"
+import ClickableTagPopover from "./ClickableTagPopover"
+import ViewInFormResponses from "./ClickableTagPopover/components/ViewInFormResponses"
+
+type Props = FormTagProps & Omit<TagProps, "colorScheme">
+
+type FormTagProps = { formId: number; guildId: number }
+
+type ClickableFormTagProps = FormTagProps & {
+  activityLogType?: ActivityLogType
+  userId?: number
+}
+
+type FormField = {
```

Comment:
We have types for these in the `@guildxyz/types` package, so let's use those! (`Schemas["Form"]` & `Schemas["Field"]`)